stable/istio-ingress-gateway/Chart.yaml

```
@@ -18,4 +18,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.5.5
+version: 2.6.5
```

Why is this `2.6.5` instead of `2.6.0`?

```suggestion
version: 2.6.0
```

---

Done!